crates/red_knot_test/src/lib.rs

```diff
@@ -15,6 +15,8 @@ mod diagnostic;
 mod matcher;
 mod parser;
 
+const MDTEST_FILE_FILTER: &str = "MDTEST_FILE_FILTER";
```

Comment:
I would rename the environment variable to `MDTEST_TEST_FILTER` because the filter isn't filtering files, but the test cases defined in a test suite (md file)